frontend/src/widgets/DeliveryMethodGraph.js

```diff
@@ -35,8 +35,13 @@ const DEFAULT_SORT_CONFIG = {
 const KEY_COLUMNS = ['Months'];
 
 export default function DeliveryMethodGraph({ data }) {
+  const exportName = useMemo(() => {
+    const TODAY = moment().format('YYYY-MM-DD');
+    return `${TODAY} Delivery Method`;
+  }, []);
+
   const widgetRef = useRef(null);
-  const capture = useMediaCapture(widgetRef, 'Total TTA hours');
+  const capture = useMediaCapture(widgetRef, exportName);
```

Comment:
Would it be cleaner to app the prefix in the useMediaCapture function, that way its done and maintained in one place. Then each of the other locations would just need the string updated to the new version?